var/spack/repos/builtin/packages/leveldb/package.py

```diff
@@ -39,21 +39,14 @@ class Leveldb(Package):
     depends_on("snappy")
 
     def install(self, spec, prefix):
-        make()
+        mkdirp(prefix.lib.pkgconfig)
 
-        mkdirp(prefix.include)
-        mkdirp(prefix.lib)
-        mkdirp(join_path(prefix.lib, 'pkgconfig'))
+        libraries  = glob.glob('out-shared/libleveldb.*')
+        libraries += glob.glob('out-static/libleveldb.*')
+        for library in libraries:
+            install(library, prefix.lib)
 
-        cp = which('cp')
-
-        # cp --preserve=links libleveldb.* prefix/lib
-        args = glob.glob('out-shared/libleveldb.*') \
-            + glob.glob('out-static/libleveldb.*')
-        args.append(prefix.lib)
-        cp('--preserve=links', *args)
```

Comment:
This option doesn't even exist on macOS. Not sure if `copy` preserves symlinks or not.